docs/references/express/token-verification.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Networkless token verification | Clerk Express.js SDK
+description: Clerk's JWT session token can be verified in a networkless manner using the JWT verification key.
+---
+
+# Networkless token verification
+
+Clerk's JWT session token can be verified in a networkless manner using the JWT verification key. By default, Clerk will use our JWKs endpoint to fetch and cache the key for any subsequent verification. If you use the `CLERK_JWT_KEY` environment variable to supply the key, Clerk will pick it up and do networkless verification for session tokens using it.
+
+To learn more about Clerk's token verification, you can find more information on our guide to [validating session tokens](/docs/backend-requests/handling/manual-jwt).
+
+The value of the JWT verification key can also be added on the instance level or on any single middleware call e.g. for Next.js.
+
+```js
+import { withAuth } from '@clerk/nextjs';
```

Comment:
⛔ This is an express guide. I think we should only use exported functionality from the package this guide refers to instead of using `@clerk/nextjs`